ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -648,6 +657,23 @@ func resourceIBMContainerVpcClusterCreate(d *schema.ResourceData, meta interface
 		}
 
 	}
+	var taints []interface{}
+	if taintRes, ok := d.GetOk("taints"); ok {
+		taints = taintRes.(*schema.Set).List()
+	}
+	if err := updateWorkerpoolTaints(d, meta, cls.ID, "default", taints); err != nil {
+		return err
+	}
+
+	if _, ok := d.GetOk("kms_config"); ok {
```

Comment:
this is now present in resourceIBMContainerVpcClusterCreate and in resourceIBMContainerVpcClusterUpdate too.
does it mean it will be executed twice in a create?